src/js/background.js

```diff
@@ -766,6 +789,51 @@ Badger.prototype = {
     chrome.browserAction.setTitle({tabId: tab.id, title: "Privacy Badger"});
   },
 
+  showCookies: function(fqdn) {
+    let domain = window.getBaseDomain(fqdn);
+    chrome.cookies.getAll({domain}, cookies => {
+      console.log('Cookies for domain: ' + domain);
+      cookies.forEach(cookie => {
+        console.log('\t -------------');
+        console.log('\t cookie name: ' + cookie.name);
```

Comment:
Following up on our call, I was wondering if we could direct users to visit a URL that gives more complete information about "local storage" (cookies, localStorage, etc.) set by a domain. Looks like we can, at least in current Chrome (may break at any point probably), for example: `chrome://settings/cookies/detail?site=www.google.com`.